charts/nri-metadata-injection/templates/admission-webhooks/job-patch/job-createSecret.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if (and (not .Values.customTLSCertificate) (not .Values.certManager.enabled)) }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name:  {{ template "nri-metadata-injection.fullname" . }}-admission-create
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+  labels:
+    app: {{ template "nri-metadata-injection.name" $ }}-admission-create
+{{ include "nri-metadata-injection.labels" $ | indent 4 }}
+spec:
+  template:
+    metadata:
+      name:  {{ template "nri-metadata-injection.fullname" . }}-admission-create
+      labels:
+        app: {{ template "nri-metadata-injection.name" $ }}-admission-create
+{{ include "nri-metadata-injection.labels" $ | indent 8 }}
+    spec:
+      containers:
+        - name: create
+          image: {{ .Values.jobImage.repository }}:{{ .Values.jobImage.tag }}
+          imagePullPolicy: {{ .Values.jobImage.pullPolicy }}
+          args:
+            - create
+            - --host={{ template "nri-metadata-injection.fullname" . }},{{ template "nri-metadata-injection.fullname" . }}.{{ .Release.Namespace }}.svc
+            - --namespace={{ .Release.Namespace }}
+            - --secret-name={{ template "nri-metadata-injection.fullname" . }}
+	        {{- if .Values.jobImage.volumeMounts }}
+	        volumeMounts:
+	        {{- include "tplvalues.render" ( dict "value" .Values.jobImage.volumeMounts "context" $ ) | nindent 10 }}
+	        {{- end }}
```

Comment:
Tabs here

Reply:
Thanks, fixed. I need to fix my vim config finally..